packages/babel-traverse/test/fixtures/conversion/this-source-maps/source-map.json

```diff
@@ -13,5 +13,6 @@
   "sourcesContent": [
     "function fn() {\n  var inner = () => {\n    console.log(this);\n  };\n}"
   ],
-  "mappings": "AAAA,SAASA,EAAEA,CAAA,EAAG;EAAA,IAAAC,KAAA;EACZ,IAAIC,KAAK,GAAG,SAAAA,CAAA,EAAM;IAChBC,OAAO,CAACC,GAAG,CAACH,KAAI,CAAC;EACnB,CAAC;AACH"
+  "mappings": "AAAA,SAASA,EAAEA,CAAA,EAAG;EAAA,IAAAC,KAAA;EACZ,IAAIC,KAAK,GAAG,SAAAA,CAAA,EAAM;IAChBC,OAAO,CAACC,GAAG,CAACH,KAAI,CAAC;EACnB,CAAC;AACH",
+  "ignoreList": []
```

Comment:
In the future, we can add injected inline helper functions to this `ignoreList` so that they are skipped by debuggers.